go.mod

```diff
@@ -28,6 +28,8 @@ require (
 	github.com/go-ini/ini v1.62.0 // indirect
 	github.com/go-openapi/spec v0.19.3
 	github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0 // indirect
+	github.com/googleapis/gnostic v0.3.0 // indirect
```

Comment:
Please check if these are really needed.

Reply:
Yes, output of `go mod graph`:

```
github.com/fission/fission github.com/googleapis/gnostic@v0.3.0
k8s.io/apimachinery@v0.17.2 github.com/googleapis/gnostic@v0.0.0-20170729233727-0c5108395e2d
k8s.io/kube-openapi@v0.0.0-20191107075043-30be4d16710a github.com/googleapis/gnostic@v0.0.0-20170729233727-0c5108395e2d
k8s.io/apiserver@v0.17.2 github.com/googleapis/gnostic@v0.0.0-20170729233727-0c5108395e2d
k8s.io/apiextensions-apiserver@v0.17.2 github.com/googleapis/gnostic@v0.0.0-20170729233727-0c5108395e2d
k8s.io/client-go@v0.17.2 github.com/googleapis/gnostic@v0.0.0-20170729233727-0c5108395e2d
```